src/library_html5.js

```diff
@@ -497,7 +505,15 @@ var LibraryHTML5 = {
       || (e.screenY-JSEvents.previousScreenY)
 #endif
       ;
+#if MEMORY64
+    {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.movementX, 'movementXValue', 'i64') }}};
+    {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.movementY, 'movementYValue', 'i64') }}};
+#else
+    {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.movementX, 'movementXValue', 'i32') }}};
+    {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.movementY, 'movementYValue', 'i32') }}};
```

Comment:
Again I would just `*` here instead of the if/else block.

Reply:
Done.